client/src/components/Menu/Menu.scss

```diff
@@ -771,6 +771,10 @@
       .cms-help__badge {
         display: block;
       }
+
+      .cms-help__logo {
+        min-width: 26px;
```

Comment:
What is the purpose of this line? Nothing changes when I disable it.

Reply:
I had issues when cwp-core was installed with it squishing the logo

That line was previously in the custom.css file in cwp-core, that now that's been deprecated seems like it makes more sense here.

Since it possibly has no effect, it has no real downside to it being here

Reply:
You had extra CSS rule in CWP-core which have now been removed. Keeping dead-weight code around for nothing just increases our technical debt.

Take the rule out.